### PR TITLE
IBX-5107: Added REST API for Product Catalog's aggregations

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://github.com/ibexa/product-catalog/pull/932
| Versions      | master (4.5)
| Edition       | all

#### Checklist

- [x] Text renders correctly - generate HTML
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductView.json.example [hunk 1]
```diff
@@ -7,96 +7,53 @@
...
-                                        "value": "#000000",
-                                        "group": "Test Attribute group",
-                                        "position": 0,
-                                        "options": []
-                                    }
-                                }
-                            ]
+                            "identifier": "desk",
+                            "name": "Desk",
+                            "AttributeAssignmentList": []
```
Comment (Steveb-p) [inline docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductView.json.example:24]: Attribute assignment lists are suddenly empty in examples. I assume it was not intentonal?

Comment (DominikaK) [reply docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductView.json.example:24]: Yes, bad copying, will fix

File: docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductViewInput.json.example [hunk 2]
```diff
@@ -5,13 +5,23 @@
...
                 "ProductName": "descending"
-            }
+            },
+            "Aggregations": [
+                {
+                    "ProductPriceRange": {
+                        "name": "price",
+                        "currencyCode": "EUR",
+                        "ranges": [
+                            {"from": 0, "to": 10000}
```
Comment (Steveb-p) [inline docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/POST/ProductViewInput.json.example:20]:
```suggestion
                            {"from": 0, "to": 10000},
                            {"from": 10000, "to": null}
```

**Review by Steveb-p — APPROVED** [review]
The size of the other two file changes worries me, but it seems to be generated from an actual instance?

**Comment by DominikaK** [comment]
Yep, examples are all generated.

**Pull request merged**